server/routes/bookedtoday/choosePrisonerController.ts

```diff
@@ -13,4 +15,37 @@ export default class ChoosePrisonerController {
       })
     }
   }
+
+  private handleNewPrisoner(arrival: Arrival, res: Response): void | PromiseLike<void> {
+    return !arrival.prisonNumber && !arrival.pncNumber
+      ? res.redirect(`/prisoners/${arrival.id}/search-for-existing-record/new`)
+      : /**
+         * TODO 3 situations to deal with, will redirect to new pages for each of:
+         * - no matches         - show "This person doesn't have a record"
```

Comment:
if an identifier is provided then we assume that the data is accurate (but give an option to bail out)